extensions/amp-ad/0.1/amp-ad-xorigin-iframe-handler.js

```diff
@@ -118,15 +117,12 @@ export class AmpAdXOriginIframeHandler {
         () => this.sendEmbedInfo_(this.baseInstance_.isInViewport()));
 
     // To provide position to inabox.
-    if (isExperimentOn(this.win_, 'inabox-position-api')) {
```

Comment:
What is this change about?

Reply:
Do we only want to do this if we're in the experiment?